v2/controllers/pod_watcher.go

```diff
@@ -171,7 +171,7 @@ OUTER2:
 				continue OUTER2
 			}
 		}
-
+		logger.Info("delete peer", "caller", "addPod", "pod", pod.Namespace+"/"+pod.Name, "ip", eip.String())
```

Comment:
Could you log key, existing, podIPs as well? I want to know the state in detail when the delete peer happens.

Reply:
I added logging fields `podIPs` and `existing` and changed a key name from `pod` to `key`.

Reply:
https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/controller/controller.go#L118

Reply:
So, do we have not to add key(`pod.Namespace+"/"+pod.Name`) in this case?

Reply:
Yes, the controller passes a pre-configured logger to the reconciler so that every log line includes the resource name and namespace.